src/Raven.Client/Constants.cs

```diff
@@ -494,5 +494,9 @@ private Obsolete()
 
             public const string GraphApi = "Graph API will be removed in next major version of the product.";
         }
+        public static class Collections
+        {
+            public const string AllCollection = "@all";
```

Comment:
This shouldn't be on the client. Can you use `AllDocumentsCollection` instead?

Reply:
Done. AllCollection removed